### ISSUE TITLE
Is it possible to use kstone to sync one etcd clusters to another

### ISSUE THREAD

**Issue by Tendrun101** [tkestack/kstone#107]
There two etcd clusters in two data center (same city)，the first one as  primary, another as backup. 
If the primary cluster fails, we can use backup cluster.  So we need to sync between the two clusters

**Comment by tangcong** [comment]
yes, it is a useful feature and we will implement it in the future.
